README.md

```diff
@@ -1,11 +1,106 @@
-# amrita_pyq
+# **Amrita PYQ CLI Tool Documentation**
 
-## Contributing
+## **Overview**
+The Amrita PYQ CLI Tool is a command-line interface application that simplifies accessing previous year question papers (PYQs) by directly fetching and displaying them in your default browser.
 
-We welcome contributions! Please see our [Contributing Guide](.github/CONTRIBUTING.md) for more details.
+---
 
-<!-- 
-## Code of Conduct
+## **Prerequisites**
+Ensure the following requirements are met before using the application:
 
-This project adheres to a [Code of Conduct](CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code.
--->
+- **Network**: You must be connected to the **Amrita WiFi** or use a **VPN** to access the network.  
+- **Development Environment**:  
+  - **Golang** must be installed on your system.  
+  - Ensure a working **Go Compiler** is set up.
+
+---
+
+## **Usage Instructions**
+
+### Step 1: Connect to Amrita WiFi or VPN
+Before using the tool, ensure that you are connected to **Amrita WiFi** or using a **VPN** to access the network. This is mandatory for fetching the previous year question papers.
+
+### Step 2: Clone the Repository
+```bash
+git clone https://github.com/CSE-25/amrita_pyq
+```
+
+---
+
+### Step 3: Run the Application
+1. Open the `main.go` file in your preferred code editor (eg:VS Code).  
+2. Execute the application:
+   ```bash
+   go run main.go
+   ```
+3. A menu will appear with options to choose from.
+
+### Output
+![image](https://github.com/user-attachments/assets/862562c1-4db7-4200-82d2-07f58f6fad09)
+
+---
+
+### Step 4: Select an Option
+1. Use the menu to choose an option.  
+2. The tool will process your request and fetch the desired question paper.
+
+### Output
+![image](https://github.com/user-attachments/assets/0eb76815-cfd8-4f48-8c67-bec55bc65f11)
+
+---
+
+
+### Step 5: View the Question Paper
+1. The question paper will automatically open as a **PDF** in your default web browser.
+
+### Output
+![image](https://github.com/user-attachments/assets/d926aad6-2adc-41ce-a1b4-c287f1ccab1d)
+![image](https://github.com/user-attachments/assets/62e4f6cb-2d38-41ec-9e00-2e1d347114e2)
```

Comment:
Can you replace this image with a screenshot of it opening in a browser?

Reply:
Could you clarify whether you're referring to the image mentioned at Line 57 or Line 58, or both?

Reply:
58

Reply:
did the changes, please check it out

Reply:
@Abhinav-ark